code/modules/mob/living/carbon/human/species/station/prometheans.dm

```diff
@@ -39,7 +39,7 @@ var/datum/species/shapeshifter/promethean/prometheans
 	virus_immune =	1
 	blood_volume =	560
 	min_age =		1
-	max_age =		5
+	max_age =		10
```

Comment:
Has the wiki's timeline been adjusted for this?

Reply:
This is the reason for DNM

Reply:
Timeline updated.